File: Python_Ruby/function/5.py
def login(_id):
    members = ['egoing', 'k8805', 'leezche']
    
    for member in members:
        if member == _id:
            return True
    
    return False

input_id = input("Please input your id\n")

if(login(input_id)):
    print('Hello, ' + input_id)
else:
    print('Who are you?')

File: Python_Ruby/function/5.rb
def login(id)
    members = ['egoing', 'k8805', 'leezche']
    for member in members do
        if member == id
            return true
        end
    end
    return false
end

puts("Please input your password")
input_id = gets.chomp()

if(login(input_id))
    puts('Hello, ' + input_id)
else
    puts('Who are you?')
end